challenges/challenge-1/challenge-1.html

```diff
@@ -249,6 +249,8 @@ <h1>Rizal M</h1>
             <p>
               <q>Learn something useful and spread it</q>
               <br>Anonymous
+            </p>
```

Comment:
Thank you.  I think that I may be the one that deleted these lines when resolving a conflict.  I am so glad you caught it.